calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -0,0 +1,309 @@
+#!/bin/python
+from collections import namedtuple
+import json
+import os
+import socket
+import sys
+from subprocess import check_output, CalledProcessError, check_call
+import requests
+import sh
+
+# Append to existing env, to avoid losing PATH etc.
+# Need to edit the path here since calicoctl loads client on import.
+# TODO-PAT: This shouldn't be hardcoded
+os.environ['ETCD_AUTHORITY'] = 'kubernetes-master:6666'
+
+from calicoctl import container_add
+from pycalico.datastore import IF_PREFIX
+from pycalico.util import generate_cali_interface_name
+
+CALICOCTL_PATH = os.environ.get('CALICOCTL_PATH', '/home/vagrant/calicoctl')
+calicoctl = sh.Command(CALICOCTL_PATH).bake(_env=os.environ)
+
+ETCD_AUTHORITY_ENV = "ETCD_AUTHORITY"
+PROFILE_LABEL = 'CALICO_PROFILE'
+ETCD_PROFILE_PATH = '/calico/'
+AllowRule = namedtuple('AllowRule', ['port', 'proto', 'source'])
+
+
+class NetworkPlugin(object):
+    def __init__(self):
+        self.pod_name = None
+        self.docker_id = None
+
+    def create(self, args):
+        """"Create a pod."""
+        # Calicoctl only
+        self.pod_name = args[3].replace('-', '_')
+        self.docker_id = args[4]
+
+        print('Configuring docker container %s' % self.docker_id)
```

Comment:
Yep, they will be logged at level 5 by kubelet (--v=5 as an argument to kubelet to allow them to be logged).

I was planning to refactor this to use the logging module, and just set stdout as the stream handler for the k8s plugin. TODO in a later PR.